tests/shared/testRefForwarding.tsx

```
@@ -1,18 +1,23 @@
-import { createRef } from 'react'
+import { cloneElement, createRef } from 'react'
 import { renderWithNexUIProvider } from './renderWithProvider'
-import type { ElementType } from 'react'
+import type { ReactElement } from 'react'
 import type { RenderWithNexUIProviderOptions } from './renderWithProvider'
 
 export function testRefForwarding(
-  Component: ElementType,
+  component: ReactElement<{ ref?: React.Ref<unknown> }>,
   constructor?: Function,
```

Biome (2.1.2)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)